tests/zfs-tests/tests/functional/cli_root/zfs_set/property_alias_001_pos.ksh

```diff
@@ -22,13 +22,11 @@
 
 #
 # Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
```

Comment:
another duplicated space

Reply:
Don't forget the comma's before inc if we are nitting